src/Essentials/src/Geolocation/LocationEventArgs.shared.cs

```diff
@@ -0,0 +1,18 @@
+﻿#nullable enable
+using System;
+
+namespace Microsoft.Maui.Devices.Sensors
+{
+	public class LocationEventArgs : EventArgs
+	{
+		public Location Location { get; }
+
+		public LocationEventArgs(Location location)
+		{
+			if (location == null)
+				throw new ArgumentNullException(nameof(location));
```

Comment:
```suggestion
			ArgumentNullException.ThrowIfNull(nameof(location));
```

Reply:
`ArgumentNullException.ThrowIfNull()` is not available with `netstandard2.0` and `netstandard2.1`, so I can't replace the call. I wonder if `location` can ever be `null` with the `#nullable enable` at the start of the file.